templates/cluster/remote-cluster/templates/machines.yaml

```diff
@@ -19,6 +19,8 @@ apiVersion: bootstrap.cluster.x-k8s.io/v1beta1
 kind: K0sWorkerConfig
 metadata:
   name: {{ include "cluster.name" $ }}-{{ $index }}
+  annotations:
+    helm.sh/resource-policy: keep
```

Comment:
coulld you quickly elaborate on why do we need the explicit anno for the helm?

Reply:
Sure.

Reply:
I see, very subtle nuance. Thanks for the clarification